code/game/antagonist/station/traitor.dm

```diff
@@ -57,8 +57,8 @@
 	only_human = FALSE
 
 	possible_objectives = list(
-	/datum/objective/assasinate = 100,
-	/datum/objective/assasinate = 90,
+	/datum/objective/assassinate = 100,
+	/datum/objective/assassinate = 90,
```

Comment:
That's not target of PR, but on of lines above is redundant.

Reply:
I'm not 100% certain, but i think that's a 90% chance of getting a second target in addition to the first

Reply:
no you're right it is redundant, fixing